pages/linux/caja.md

```diff
@@ -0,0 +1,24 @@
+# caja
+
+> Manages files and directories in MATE desktop environment.
+> More information: <https://manned.org/caja.1>.
+
+- Open a current user home directory:
+
+`caja`
+
+- Open specific directories in separate windows:
+
+`caja {{path/to/directory{1,2,...}}}`
```

Comment:
Mixing this example with specific bash syntax is hard to understand for new users
```suggestion
`caja {{path/to/directory1 path/to/directory12 ...}}}`
```

Reply:
I don't understand. In some pages it is required to use brace expansion and in others - not. We have to describe a formal rule when to use it.

Reply:
#7969 - please consider this issue